build/vsts_build.yaml

```diff
@@ -549,6 +549,7 @@ phases:
     FullVstsBuildNumber: $[dependencies.Initialize_Build.outputs['updatebuildnumber.FullVstsBuildNumber']]
   condition: "and(succeeded(),eq(variables['RunEndToEndTests'], 'true')) "  	
   queue:
+    timeoutInMinutes: 90
```

Comment:
I assume we just had the default which was 60 mins?

Reply:
yep.